README.md

```diff
@@ -41,5 +43,10 @@ A demo can be run with
 ```
 npm run demo
 ```
-
 and going to `http://localhost:1234` in the browser.
+
+*NOTE: You should have enough test ether to open channel, default value hardcoded in `profile.json` is 100 eth.*
```

Comment:
This is inaccurate. The value is specified by the `amountWei` parameter in the `openEthChannel` API.

Reply:
Fixed

Reply:
Sorry. What I meant was that it's not hardcoded in `profile.json`, but passed as the `amountWei` parameter. Are you talking about the example in `demo/demo.js`?

Reply:
Fixed, I was in hurry ;-)